src/components/apiDelegation/offered/OverviewPage/OverviewPage.tsx

```diff
@@ -1,19 +1,21 @@
-import { Page, PageHeader, PageContent } from '@altinn/altinn-design-system';
+import { Page, PageHeader, PageContent, PageSize } from '@altinn/altinn-design-system';
 import { useTranslation } from 'react-i18next';
 import * as React from 'react';
 
 import { ReactComponent as ApiIcon } from '@/assets/ShakeHands.svg';
 import { PageContainer } from '@/components/reusables/PageContainer';
+import { useMediaQuery } from '@/resources/hooks';
 
 import { OverviewPageContent } from '../../reusables/OverviewPageContent';
 import { LayoutState } from '../../reusables/LayoutState';
 
 export const OverviewPage = () => {
   const { t } = useTranslation('common');
+  const isSm = useMediaQuery('(max-width: 768px)');
 
   return (
     <PageContainer>
-      <Page>
+      <Page size={isSm ? PageSize.Small : PageSize.Medium}>
         <PageHeader icon={<ApiIcon />}>{t('api_delegation.api_delegations')}</PageHeader>
```

Comment:
The icon in the page header looks a bit too shrunken on mobile screens
Not sure if this is a design-system issue or something that can be fixed here but it should probably be addressed somewhere.
![image](https://user-images.githubusercontent.com/22012997/224715902-e991c810-da7b-4a91-a37a-91d270299454.png)


Maybe a solution could be to simply drop the icon on small screens?
It does make the header title look a bit nicer imo 
![image](https://user-images.githubusercontent.com/22012997/224715498-1547b1f0-b4c0-4d97-8c7a-1d9a7af61e62.png)

Reply:
This is really a bug with long texts in AccordionHeader. The bug don't exist on the other pages as there are shorter texts in AccordionHeader there. I think I will keep is at it is for now and fix the root to the problem when I'm going to implement Page component in digdir design system. Marianne is working on speccing Page so it will soon be ready.